utils/checkForLastLogin.js

```diff
@@ -0,0 +1,29 @@
+const CronJob = require('cron').CronJob
```

Comment:
I think it might be better to create a folder with cron-jobs, not utils

Reply:
done